README.md

```diff
@@ -573,6 +573,49 @@ Don't worry about `adbkey` or `adbkey.pub` under `/.android`, not required.
 
   > Unfortunately it is not possible to pass through a USB device (or a serial port) to a container.
 
+## Firebase Test Lab 
+You can also run your UI tests on Firebase Test Lab using emulators and physical devices, to do so first you need to
+create a service account with an “editor” role in the [Google Cloud Platform](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts?pli=1&supportedpurview=project) 
+and enable [Google Cloud Testing API and Google Cloud Results API](https://console.developers.google.com/apis/library?pli=1). Later, in your Google Cloud Console go to “Credentials” > “Create Credentials” > “Create Service Account Key” fill the data and select key type as JSON.
+
+
+* Now let's pull and run the Docker image with Google Cloud SDK
+
+```bash
+# pull the image
+docker pull thyrlian/android-firebase-test-lab
+# run the container passing the service account Json (at least the first time) as external volume
+docker run -d -p 2222:22 -v $(pwd)/sdk:/opt/android-sdk -v /path/on/host/auth.json:/auth.json thyrlian/android-firebase-test-lab
+```
+* Log in and run your tests
+
+```bash
+# check the running process and get a interactive shell
+Docker ps
+Docker exec -it <container-id> /bin/bash
+
+# check Google Cloud is installed properly
+gcloud info --format="value(installation.sdk_root)"
```

Comment:
This would be useful for testing and debugging.  I suggest moving this to a dedicated `version_inspector.sh` file inside the `firebase_test_lab` directory, and build this image, run this script with TravisCI (just add to `.travis.yml` configuration).